portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase

```diff
@@ -1060,7 +1060,6 @@ ${company.getKeyObj()}
 
 			var templateScript = '''
 ${objectUtil("com.liferay.portal.service.UserLocalServiceUtil").getService()}<br />
```

Comment:
@vicnate5 Hi Victor,
I'm not very sure what to do in this testcase. I removed utilLocator line. 
But notice that full name of UserLocalServiceUtil does not exist, 
it should be "com.liferay.portal.**kernel**.service.UserLocalServiceUtil"

Not sure if this is a problem since I'm not familiar with testcase syntax